index.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express')
+const bodyparser = require('body-parser')
+const { getAllVillains, getVillainBySlug, addVillain } = require('./controllers/villains')
+
+const app = express()
+
+app.get('/villains', getAllVillains)
+
+app.get('/villains/:slug', getVillainBySlug)
```

Comment:
params should match with params used in the controller
should be:
app.get('/villains/:searchTerm', getVillainBySlug)
instead of:
app.get('/villains/:slug', getVillainBySlug)